### DISCUSSION TITLE
Can you capture the output of cspell-action?

### DISCUSSION THREAD

**Discussion by Nightblade** [streetsidesoftware/cspell-action#872] [Q&A] [answered]
I would like to add a pretty summary that includes the output of `cspell-action` to my spellcheck workflow.  Is this even possible?  I'm a bit of an actions/yaml noob...

**Comment by Jason3S** [comment, marked as answer]
@Nightblade,

At the moment, it is not possible, but you make a good point.

What would you want in the output?

**Reply by Nightblade** [reply]
Howdy!  Now that I've played around with it a bit more, I'm not 100% sure this is necessary.  And my brain is a bit fried atm, so I can't remember exactly what I was trying to do!  Maybe delete this and if I remember I'll make a fresh discussion?  Thanks.
